pandas_datareader/google/options.py

```diff
@@ -0,0 +1,193 @@
+import datetime as dt
+import re
+import json
+
+from pandas import DataFrame, MultiIndex
+
+from pandas_datareader.base import _BaseReader
+
+
+class Options(_BaseReader):
+    """
+    ***Experimental***
+    This class fetches call/put data for a given stock/expiry month.
+
+    It is instantiated with a string representing the ticker symbol.
+
+    The class has the following methods:
+        get_options_data(month, year, expiry)
+        get_call_data(month, year, expiry)
+        get_put_data(month, year, expiry)
+        get_near_stock_price(opt_frame, above_below)
+        get_all_data(call, put)
+        get_forward_data(months, call, put) (deprecated)
+
+    Examples
+    --------
+    # Instantiate object with ticker
+    >>> aapl = Options('aapl')
+
+    # Fetch next expiry call data
+    >>> calls = aapl.get_call_data()
+
+    # Can now access aapl.calls instance variable
+    >>> aapl.calls
+
+    # Fetch next expiry put data
+    >>> puts = aapl.get_put_data()
+
+    # Can now access aapl.puts instance variable
+    >>> aapl.puts
+
+    # cut down the call data to be 3 below and 3 above the stock price.
+    >>> cut_calls = aapl.get_near_stock_price(call=True, above_below=3)
+
+    # Fetch call and put data with expiry from now to 8 months out
+    >>> forward_data = aapl.get_forward_data(8, call=True, put=True)
+
+    # Fetch all call and put data
+    >>> all_data = aapl.get_all_data()
+    """
+
+    _OPTIONS_BASE_URL = "http://www.google.com/finance/option_chain?q={sym}" \
+                        "&expd={day}&expm={month}&expy={year}&output=json"
+
+    def __init__(self, symbol, session=None):
+        """ Instantiates options_data with a ticker saved as symbol """
+        self.symbol = symbol.upper()
+        super(Options, self).__init__(symbols=symbol, session=session)
+
+    def get_options_data(self, month=None, year=None, expiry=None):
```

Comment:
This goes back to the conversation on the API. With weekly options, it is not really clear where to map YEAR/MONTH (to 3rd Fri of the month?) ... I'm open to map them as suggested.